apiserver/facades/client/application/application.go

```diff
@@ -1596,6 +1601,14 @@ func (api *APIBase) ScaleApplications(args params.ScaleApplicationsParams) (para
 		} else if err != nil {
 			return nil, errors.Trace(err)
 		}
+		if arg.ScaleChange != 0 {
+			currentScale := app.GetScale()
+			newScale := currentScale + arg.ScaleChange
```

Comment:
It's not quite this simple as we should add an assert that the current scale hasn't changed. This functionality needs to be moved to state where an assert can be used. ie introduce a ChangeScale() method.